src/styles/console.m.css

```diff
@@ -0,0 +1,47 @@
+.root {
+	display: inherit;
+}
+
+.consoleRow {
```

Comment:
having totally empty styles does cause issues with compilation... so I would at least put `display: inherit;` in these...

Reply:
Addressed!